src/Krank/Checkers/IssueTracker.hs

```diff
@@ -22,10 +24,15 @@ githubRE = do
   few (psym ('/' /=))
   "/"
   "issues/"
-  issueNum <- few (psym isDigit)
+  issueNum <- some (psym isDigit)
   optional "/"
+  " " <|> "\t" <|> "\n"
   return $ read issueNum
 
 check :: FilePath
-      -> [Violation]
-check file = []
+      -> IO [Int]
+check file = do
+  content <- readFile file
+  print content
```

Comment:
Is that a debug leftover?

Reply:
yeah. as I said, PR not ready, but there to start a discussion ;)